src/machine/asm/execute_aarch64.S

```diff
@@ -242,9 +244,10 @@ _ckb_vm_x64_execute:
 .globl ckb_vm_x64_execute
 ckb_vm_x64_execute:
 #endif
-  stp x19, x20, [sp, -48]!
+  stp x19, x20, [sp, -64]!
   stp x21, x22, [sp, 16]
   stp x23, x30, [sp, 32]
+  stp x24, x25, [sp, 48]
```

Comment:
Notice we are not using x30 in this file at all, can we replace x30 with x24 and see if it works?

Reply:
Yes is works. We can using ohter registers to store some memory data.